core/core-api/src/main/java/io/fabric8/launcher/core/api/readme/ReadmeProcessor.java

```diff
@@ -3,6 +3,8 @@
 import java.io.IOException;
 import java.util.Map;
 
+import javax.annotation.Nullable;
```

Comment:
Can this interface be moved to the documentation package?

Reply:
I though it was already there.. lol.. fixed